### DISCUSSION TITLE
Converte HTML to docx

### DISCUSSION THREAD

**Discussion by youssefelya** [dolanmiu/docx#2689] [General]
Hello,
I would like to inquire if it is possible to achieve the following:
I have a typescript application where I am fetching data from a database and generating a Word document using the docx library. The challenge is that I have a field containing HTML code, such as < p >paragraph with < a href='https://example.com' >a  link< /a>.< /p >. I would like to know if it is possible to convert this HTML into a Word document so that if the text is formatted as strong/bold in HTML, it will appear the same (strong/bold) in the Word document.

Thank you.

**Comment by anti-the-social** [comment]
Hi!

Rules of html&css styling are not fully compatible with .docx files 
Example of such would be text floating around an image.

There are many project on the market who tries to create html editors and convert their created html into docx with semi-successful results.

This project isn't about it tho, mode like a tool to create with JS commands some sort of .docx file in return (.zip with set of .xml files according to OOXML stardard)
